### PR TITLE
LPS-185898 Hide oldValue function in readOnly Expression Builder and Make readOnly property not required

### PR DESCRIPTION
**Pull request by p-albuquerque**
Related issue: https://issues.liferay.com/browse/LPS-185898

### CODE REVIEW

**Review by cgmonte — APPROVED** [review]
LGTM!

**Comment by p-albuquerque** [comment]
ci:test:sf

**Comment by p-albuquerque** [comment]
ci:test:relevant

**Comment by guilhermedcamacho** [comment]
Just started reviewing :)

**Comment by p-albuquerque** [comment]
ci:test:relevant

**Comment by p-albuquerque** [comment]
ci:test:relevant

**Comment by p-albuquerque** [comment]
ci:test:object

File: modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/display/context/util/ObjectCodeEditorUtil.java [hunk 1]
```diff
@@ -314,12 +316,17 @@ private enum DDMExpressionFunction {
 				"that-can-be-used-with-other-validation-functions",
 			"sum");
 
-		public static List<Map<String, String>> getItems(Locale locale) {
+		public static List<Map<String, String>> getItems(
+			String excludedDDMExpressionFunctionKey, Locale locale) {
```
Comment (guilhermedcamacho) [inline modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/display/context/util/ObjectCodeEditorUtil.java:320]: Can we change this method to make it more flexible?
For example, what if we want to hide more than one function?

**Review by guilhermedcamacho — CHANGES_REQUESTED** [review]

**Comment by guilhermedcamacho** [comment]
See #2223

**Pull request closed without merging**